src/languages/lib/css-shared.js

```diff
@@ -115,7 +115,48 @@ export const TAGS = [
   'tr',
   'ul',
   'var',
-  'video'
+  'video',
+  'defs',
```

Comment:
Much better! Thanks.  Is everything from here down SVG specific?

Reply:
Since you didn't mix them in yet I'm wondering if we should keep them separately, but two lists...

```js
HTML_TAGS = []
SVG_TAGS = []
TAGS = [
  ...HTML_TAGS, 
  ...SVG_TAGS
]
```

Thoughts?

Reply:
You are right.
I will edit the file and commit it again. 
Thanks for your advice

Reply:
I have committed the edited file. 
Is that as you want or not?